lab-ron/lib/greet.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
```

Comment:
This works fine, though you shouldn't have any files other than your main file (index.js, here) reference process.argv[2].  I suggest removing the greeting.js file and copy/pasting it into this file instead.